behind-the-veil-siteroot/imports/ui/components/pages/profile/ArtistProfilePage.jsx

```diff
@@ -0,0 +1,148 @@
+            <div>
+              <img
+                className="h-auto max-w-full rounded-lg object-cover object-center "
+                src="https://docs.material-tailwind.com/img/team-3.jpg"
+                alt="gallery-photo"
+              />
+            </div>
+          </div>
+          <div className="grid gap-4">
+            <div>
+              <img
+                className="h-auto max-w-full rounded-lg object-cover object-center"
+                src="https://images.unsplash.com/photo-1493246507139-91e8fad9978e?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2940&q=80"
+                alt="gallery-photo"
+              />
+            </div>
+            <div>
+              <img
+                className="h-auto max-w-full rounded-lg object-cover object-center "
+                src="https://docs.material-tailwind.com/img/team-3.jpg"
+                alt="gallery-photo"
+              />
+            </div>
+            <div>
+              <img
+                className="h-auto max-w-full rounded-lg object-cover object-center"
+                src="https://images.unsplash.com/photo-1552960562-daf630e9278b?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
+                alt="gallery-photo"
+              />
+            </div>
+          </div>
+          <div className="grid gap-4">
+            <div>
+              <img
+                className="h-auto max-w-full rounded-lg object-cover object-center"
+                src="https://images.unsplash.com/photo-1552960562-daf630e9278b?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
+                alt="gallery-photo"
+              />
+            </div>
+            <div>
+              <img
+                className="h-auto max-w-full rounded-lg object-cover object-center"
+                src="https://images.unsplash.com/photo-1629367494173-c78a56567877?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=927&q=80"
+                alt="gallery-photo"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+
+  return (
+    <WhiteBackground pageLayout={PageLayout.LARGE_CENTER}>
+      <div className="flex justify-end">
+        <Button className="flex"> {gearIcon} Settings</Button>
+      </div>
+      <ProfilePhoto className="flex container mx-auto" />
+      <div className="text-center main-text">Name</div>
+      <div className="text-center main-text">Tag</div>
+      <Tabs
+        tabs={["Dashboard", "Bookings", "My Services", "Gallery", "Reviews"]}
+        tabPanels={[
+          "Dashboard Panel",
+          "Bookings Panel",
+          "My Services Panel",
+          galleryTab,
+          "Reviews Panel",
+        ]}
+        tabsClassName="flex justify-between"
+        tabPanelsClassName="relative flex justify-around"
```

Comment:
Do you know if all the other tab panels require justify-around? If not, then I think it's better to move 'relative flex justify-around' to the outer div of galleryTab, otherwise justify-around will be unnecessarily applied to all tab panels

Reply:
Other tab panels so far does not require justify-around. Moved this as per feedback to the outer div of galleryTab! :)

Issue should be resolved.